_studio/mfx_lib/encode_hw/vp9/include/mfx_vp9_encode_hw_utils.h

```diff
@@ -435,6 +435,9 @@ template <typename T> mfxStatus RemoveExtBuffer(T & par, mfxU32 id)
     public:
         MfxFrameAllocResponse();
 
+        MfxFrameAllocResponse(MfxFrameAllocResponse const &) = delete;
+        MfxFrameAllocResponse & operator =(MfxFrameAllocResponse const &) = delete;
```

Comment:
Let's follow rule of five and define move ctor and move assignment explicitly as well (I believe they should also be deleted)

Reply:
Okay, I will add them too.
and I guess the same should be done for other encoders?

Reply:
Well, ideally - yes :) But not as part of this PR, because this one is dedicated to vp9e. Idea is that if you changing some code it is time to fix problems that you see in it. But it is up to you whether you decide to continue patching other classes